core/templates/dev/head/css/oppia.css

```diff
@@ -652,24 +696,72 @@ textarea {
   max-width: 800px;
   padding: 30px 80px;
 }
-.oppia-activity-summary-tile a,
-.oppia-activity-summary-tile-mobile a {
-  color: inherit;
-  text-decoration: none;
-}
-.oppia-dashboard-container a {
-  color: black;
-}
+
 .oppia-share-dropdown-menu li > a {
   font-weight: bolder;
   padding: 10px;
   text-align: center;
 }
-.oppia-navbar-nav > li > a {
+.modal-header {
+  border-bottom: 1px solid #e5e5e5;
+  min-height: 16.42857143px;
+  padding: 15px;
+}
+.oppia-share-publish-modal > .modal-header {
+  align-items: center;
+  flex-direction: column;
+}
+.oppia-navbar-nav {
+  float: left;
+}
+.oppia-navbar-nav > .nav-link {
   padding: 10px 5px;
 }
-.oppia-navbar-dropdown > li > a {
+.oppia-same-row-container {
+  float: left;
+}
+.oppia-navbar-dropdown .nav-link {
   color: #009688;
+}
+ .oppia-navbar-dropdown .nav-link:hover {
+  color: #888;
+}
+.oppia-activity-summary-tile a:focus,
+.oppia-activity-summary-tile-mobile a:focus {
+  box-shadow: 0 6px 13px rgba(0,0,0,0.17), 0 4px 6px rgba(0,0,0,0.23);
+}
+
+.oppia-activity-summary-tile a:focus .title-section-mask {
+  background-color: #eee;
+  height: 100%;
+  left: 0;
+  opacity: 0.4;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: 10;
+}
+.oppia-navbar-tabs > li:hover > a {
```

Comment:
Why is navbar stuff here rather than with the other navbar stuff?

Reply:
Done moved to correct place Thanks!